variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactoryTest.java

```diff
@@ -551,32 +552,34 @@ public void testParseInfo() {
     @Test
     public void testVariantIds() {
 
+        Set<String> emptySet = new HashSet<>();
         // test that an ID is properly handled
         String line = "1\t1000\trs123\tC\tT\t.\t.\t.";
         List<Variant> expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "C", "T"));
-        expResult.get(0).setIds(Collections.singleton("rs123"));
+        //EVA-942 - Since we ignore IDs submitted through VCF, they are no longer part of the expected result
+        //expResult.get(0).setIds(Collections.singleton("rs123"));
         List<Variant> result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
-        assertEquals(expResult.get(0).getIds(), result.get(0).getIds());
+        assertEquals(expResult.get(0).getIds(), emptySet);
```

Comment:
This should be the other way around. We should we checking that result.get(0).getIds() should be empty:

assertEquals(emptySet, result.get(0).getIds());